README.md

```diff
@@ -42,16 +42,28 @@ nais upload [flags]
 Flags:
   -a, --app string        name of your app
   -f, --file string       path to nais.yaml (default "nais.yaml")
-  -g, --group string      nexus group (default "nais")
   -p, --password string   the password
-  -r, --repo string       nexus repo (default "m2internal")
   -u, --username string   the username
   -v, --version string    version you want to upload
 ```
 
-Will upload `nais.yaml` to Nexus. If using default values, only `app`, `version`, `username` and `password` argument is required.
+Will upload `nais.yaml` to Nexus.
 
-The username and password may be specified using environment variable `NEXUS_USERNAME` and `NEXUS_PASSWORD` instead.
+The username and password may be specified using environment variable `NEXUS_USERNAME` and `NEXUS_PASSWORD`, and
+the Nexus repo can be specified using `NEXUS_URL`.
```

Comment:
Unnecessary new line.

Reply:
It's because the line would exceed 120 characters in width, but I can change back?

Reply:
Not sure, it may just be me that prefer my markdown not formatted by hand.

Reply:
I know, that's why I feel they are unnecessary. But, it may be for manually formatting that they ignore the single new line. So, it's up to you :)